tests/integration/test_iivsearch.py

```diff
@@ -8,6 +8,35 @@
 from pharmpy.tools import fit, retrieve_models, run_iivsearch
 
 
+def test_no_of_etas_keep(tmp_path, model_count, start_model):
```

Comment:
I would only have one test here since it takes a while to run the integration tests.